plugins/helper.js

```diff
@@ -3,6 +3,7 @@
 const boom = require('@hapi/boom');
 const dayjs = require('dayjs');
 const STAGE_PREFIX = 'stage@';
+const STAGE_TEARDOWN_PATTERN = /^stage@([\w-]+)(?::teardown)$/;
```

Comment:
I think this would also work, but no strong preference
```suggestion
const STAGE_TEARDOWN_PATTERN = /^stage@([\w-]+):teardown$/;
```